### PR TITLE
V2

### PR DESCRIPTION
**Pull request by aendrew**

### CODE REVIEW

File: src/models.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+interface QuestionAttributes {
+  id?: number;
+  order: number | null;
+  section: string;
+  question: string;
+  type: "range" | "multiple-choice";
+  choices: {
+    a: string | null;
+    b: string | null;
+    c: string | null;
```
Comment (tlfrd) [inline src/models.ts:19]: Could we add a fourth choice here, `d`?

File: src/api/project/controller.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+export interface BerthaQuestion {
+  section: string;
+  order: number | null;
+  question: string;
+  type: "multiple-choice" | "range";
+  choices: {
+    a: string | null;
+    b: string | null;
+    c: string | null;
```
Comment (tlfrd) [inline src/api/project/controller.ts:128]: Can we add the `d` option here too

File: src/api/project/controller.aggregate.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    });
+
+    if (!project || !project.questions) throw new Error("Project not found.");
+
+    const questions = await Promise.all(
+      project.questions.map(async (question) => {
+        // @TODO find way of doing this via the ORM, or not...
+        const answers = await sequelize.query(
+          `SELECT value, count(value), NTILE(${
+            question.type === "multiple-choice" ? 4 : 10
```
Comment (tlfrd) [inline src/api/project/controller.aggregate.ts:28]: Is this adding quartiles and deciles?

**Review by tlfrd — APPROVED** [review]
LGTM - data model looks good

File: src/api/project/controller.aggregate.ts (hunk 3 above)
Comment (aendrew) [reply src/api/project/controller.aggregate.ts:28]: Yeah, it calculates 10 bins if it's range or 4 bins if it's multiple choice.

**Pull request merged**